eoscostarica/eos-rate
# Fix/bp load

### GH Issue

Resolves #624 

### Steps to test

1. Make run
2. Check `anonymous` exist
3. Check `rateProducer` action exist
